controllers/assessments.controller.js

```diff
@@ -28,13 +28,12 @@ const createAssessment = async (req, res) => {
 };
 
 const getAssessmentById = async (req, res) => {
+    const assessmentID = req.query?.id;
```

Comment:
Let's stay consistent throughout the project and start using `assessmentId` instead of `assessmentID`